pkg/airgapped/plugin_bundle_upload.go

```diff
@@ -4,12 +4,15 @@
 package airgapped
 
 import (
-	"fmt"
 	"os"
 	"path/filepath"
 
 	"github.com/pkg/errors"
+
 	"github.com/verybluebot/tarinator-go"
+
+	"github.com/vmware-tanzu/tanzu-cli/pkg/utils"
+
 	"gopkg.in/yaml.v3"
 
 	"github.com/vmware-tanzu/tanzu-cli/pkg/carvelhelpers"
```

Comment:
nit: It is always better to combine imports of the same repository together.

Reply:
updated the imports